.github/workflows/pr-js-tests.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
 
-      - run: pnpm -- turbo run test --filter=!cli --color
+      - run: pnpm -- turbo run test  --filter=...[${{ env.GITHUB_BASE_REF }}] --filter=!cli --color
```

Comment:
`env.GITHUB_BASE_REF` is empty

https://github.com/vercel/turborepo/actions/runs/3107719853/jobs/5036137265#step:5:12

Reply:
Is that intentional?

Reply:
I believe these are not accessed from `env`
https://docs.github.com/en/actions/learn-github-actions/environment-variables#:~:text=C%3A%5Chostedtoolcache%5Cwindows-,Note%3A,read%20during%20workflow%20processing%20using%20the%20%24%7B%7B%20github.ref%20%7D%7D%20context%20property.,-Detecting%20the%20operating

Reply:
hmm, not intentional.